docs/var.tfvars-doc.md

```diff
@@ -40,17 +40,19 @@ openstack_availability_zone = ""
 These set of variables specify the cluster capacity.
 
 ```
-bastion                     = {instance_type    = "<bastion-compute-template>", image_id    = "<image-uuid-rhel>"}
-bootstrap                   = {instance_type    = "<bootstrap-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 1}
-master                      = {instance_type    = "<master-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 3}
-worker                      = {instance_type    = "<worker-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 2}
+bastion                     = {instance_type    = "<bastion-compute-template>", image_id    = "<image-uuid-rhel>", availability_zone = ""}
```

Comment:
`availability_zone = ""` actually means default AZ if I am not wrong. So it is better to not provide "" in var.tfvars file or documentation IMO.

Reply:
User can provide bastion.availability_zone if he need more customization. If only provided then we take the value.

Reply:
Done, if the **availability_zone = ""** is defined, the default availability zone will be used.